resolwe/flow/views/collection.py

```diff
@@ -56,6 +56,13 @@ class CollectionViewSet(ElasticSearchCombinedViewSet,
     }
     ordering = 'id'
 
+    def get_queryset(self):
+        """Return queryset."""
+        if self.request and self.request.query_params.get('hydrate_data', False):
+            return self.queryset.prefetch_related('entity_set')
```

Comment:
I'm appending collection_names and this looks incorrect. Should it be `'data__entity_set'`, like in entity.py/get_queryset https://github.com/genialis/resolwe/blob/784aeb2a21e6e87a398a299c4789dceb00bef951/resolwe/flow/views/entity.py#L37-L42

@hadalin / @dblenkus